config/config-example.js

```diff
@@ -379,3 +379,10 @@ exports.grouplist = [
 		ip: 's',
 	},
 ];
+
+// punishgroups - the symbols used to mark the different punishment groups, locked and muted
```

Comment:
Do we need to specify these in the Config? Presumably this needs to be dealt with in `punishments.js` anyway?